contracts/common/Registry.sol

```diff
@@ -61,10 +61,6 @@ contract Registry is Ownable {
       _rootToken != address(0x0) && _childToken != address(0x0),
       "INVALID_TOKEN_ADDRESS"
     );
-    require(
```

Comment:
?

Reply:
Removed it because it is ACLed on `onlyOwner` and I would like to have the ability to remap a token 😃